webhook/resources/backup/mutator.go

```diff
@@ -64,14 +64,11 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 		err := errors.Wrapf(err, "cannot find the backup volume label for backup %v", backup.Name)
 		return nil, werror.NewInvalidError(err.Error(), "")
 	}
-	volume, err := b.ds.GetVolumeRO(volumeName)
-	if err != nil {
-		err := errors.Wrapf(err, "failed to get volume %v", volumeName)
-		return nil, werror.NewInvalidError(err.Error(), "")
-	}
-	volumeAccessMode := volume.Spec.AccessMode
-	if volume.Spec.AccessMode == "" {
-		volumeAccessMode = longhorn.AccessModeReadWriteOnce
+	volumeAccessMode := longhorn.AccessModeReadWriteOnce
```

Comment:
The volumeAccessMode is stored in remote backup's cfg file after this [PR](https://github.com/longhorn/longhorn-manager/pull/1486).

In the backup sync case, the correct volumeAccessMode can be read from the remote cfg file. If volumeAccessMode label does not exist, we can set to `AccessModeReadWriteOnce` by default.

Reply:
What about `access mode` is different between backup cfg (RWX when backup) and volume spec (RWO now), I think it should still take the backup cfg, WDYT?

Reply:
YES, the backup will be the source of truth if it has access mode already.

Reply:
Addressed, update backup_controller.go as well.